lessons/operator/01/README.md

```diff
@@ -1,122 +1,121 @@
 
-![](img/sensu-observability-pipeline.png)
+**Example: A Sensu Event Data Structure**
+```json
+{
+  "metadata": {},
+  "entity": {},
+  "check": {},
+  "metrics": {},
+  "id": "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx",
+  "timestamp": 1234567890
+}
+```
 
-_FIGURE 2: Sensu's turn-key solution enables organizations to adopt observability pipelines with ease._
+Events can contain numeric metrics, dictionaries of metadata, labels, and annotations, as well as any other payload of information that is meaningful to your system.
+Each component in the pipeline expects a stream of events as input and outputs a stream to the next component.
 
-## Why Sensu?
+Learn more about the event data structure in the [events reference docs](https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-events/events/)
 
-What does "monitoring as code" and the "observability pipeline" have to do with eachother?
-And how is Sensu's solution better than the alternatives (including the DIY pipeline)?
-This question is answered in our [monitoring as code whitepaper](https://sensu.io/resources/whitepaper/monitoring-as-code-with-sensu) (bold added for emphasis):
+## Monitoring as Code
 
-> Everything as code initiatives as we know them are built on top of **pipelines that enable organizations to codify complex business requirements as repeatable steps**.
-> The CI pipeline (build+test) paved the way for Continuous Delivery (CD).
-> Together as the CI/CD pipeline (build+test, deploy), they provided the foundation for infrastructure as code (build+test, provision+test, deploy).
-> Like IaC, monitoring as code hooks into this CI/CD pipeline, but also requires a dedicated pipeline for ongoing observation of our systems (build+test, provision+test, deploy, observe).
-> We call this missing link the observability pipeline.
->
-> ![](img/everything-as-code.png)
+There is a strong movement towards managing all aspects of the software industry in the same way that we manage application code. 
+Collectively these are referred to as "everything-as-code" methods.
 
-Simply shoving observability data through a generic pipeline where you can cobble together disparate components to filter and route data to various backends is a great first step in an observability pipeline journey – but how to you manage the overall solution?
-And how do you onboard new users to such a bespoke solution – including teams with a multitude of disparate tools spanning multiple generations of technology?
-Sensu's turn-key observability pipeline provides a centralized control plane and end-to-end solution for monitoring as code, which enables organizations to onboard workloads to the observability pipeline in an incremental manner.
+Sensu is designed specifically to enable this kind of workflow for monitoring.
 
-![](img/technology-adoption-lifecycle.jpg)
+### Sensu is an API, Configurations are YAML
```

Comment:
s/YAML/code/ ??? 

Two reasons: not everyone loves YAML, but they're willing to tolerate it because they can do "as code" things with it. Better to refer to YAML less often in this case. Also, I personally prefer writing YAML over JSON, but Sensu supports both. Many of our customers choose JSON due to the broader availability of tooling to test/modify it. I'm sure we don't want to say "YAML or JSON" in so many places...

Reply:
Excellent point and yeah, I think using "code" here hits harder too.